rules/sinks/storages/neo4j/go.yaml

```diff
@@ -0,0 +1,12 @@
+
+# Sink Rules for storage database Neo4j Graph Database - https://neo4j.com/
+
+sinks:
+
+  - id: Storages.Neo4jGraphDatabase.ReadAndWrite
+    name: Neo4j Graph Database(ReadAndWrite)
+    domains:
+      - neo4j.com
+    patterns:
+      - "(?i)(github.com/neo4j/neo4j-go-driver).*(ExecuteQuery|ExecuteRead)"
```

Comment:
Execute Query makes sense for Read and write, but `ExecuteRead` looks like is a read operation

Reply:
Addressed here: https://github.com/Privado-Inc/privado/pull/317/commits/8d15ec0632dcb4ffc46f69741c7fea037b25da44